src/Bridge/Doctrine/Orm/ItemDataProvider.php

```diff
@@ -70,7 +71,7 @@ public function getItem(string $resourceClass, $id, string $operationName = null
         $this->addWhereForIdentifiers($identifiers, $queryBuilder);
 
         foreach ($this->itemExtensions as $extension) {
-            $extension->applyToItem($queryBuilder, $resourceClass, $identifiers, $operationName);
+            $extension->applyToItem($queryBuilder, new QueryNameGenerator(), $resourceClass, $identifiers, $operationName);
```

Comment:
The `QueryNameGenerator` should be instantiated only once before the loop and shared across all extensions. Otherwise you risk collisions.